.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224509424-eff26fc2-2096-4854-a34b-d8696c42a6d2.png)

- [ ] It would be nice if you remove your Live Demo section since you don't have any live link to make your readme look professional and clean 😊 Once you will have a live link you can add the Getting Started Section with the live link in your readme file 👍